docs/user/adding-new-metrics.md

```diff
@@ -1,7 +1,7 @@
 # Adding new metrics
 
-All metrics that your project collects must be defined in a `metrics.yaml` file. 
-This file should be at the root of the application or library module (the same directory as the `build.gradle` file you updated). 
+All metrics that your project collects must be defined in a `metrics.yaml` file.
+This file should be at the root of the application or library module (the same directory as the `build.gradle` file you updated).
```

Comment:
The `build.gradle` directory instructions seems to be very Android specific, while this part of the docs seems to be referring to all platforms.